Editor/BuildCallbacks.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if UNITY_2018_3_OR_NEWER
+﻿#if UNITY_2018_4_OR_NEWER
```

Comment:
I'd like to keep these as `2018_3_OR_NEWER` because, while we don't support all versions of 2018.3, 2019.1 doesn't include this define and this will cause compile issues